pkg/dpll/dpll.go

```diff
@@ -627,47 +628,62 @@ func (d *DpllConfig) MonitorDpll() {
 // stateDecision
 func (d *DpllConfig) stateDecision() {
 	dpllStatus := d.getWorseState(d.phaseStatus, d.frequencyStatus)
+
 	switch dpllStatus {
 	case DPLL_FREERUN, DPLL_INVALID, DPLL_UNKNOWN:
 		d.inSpec = false
 		d.sourceLost = true
-		if d.hasGNSSAsSource() && d.onHoldover {
-			d.holdoverCloseCh <- true
+		glog.Infof("on holdover %t has GNSS source %t", d.onHoldover, d.hasGNSSAsSource())
+		if d.onHoldover {
```

Comment:
we need to gate this for d.hasGNSSAsSource() ; if  this clock id has GNSS only then we control holdover

Reply:
we don't close holdover once its in holdover if other nic state change to Freerun etc

Reply:
Fixed, thanks!